### PR TITLE
add RasterVectorJoin Operator

### PR DESCRIPTION
**Pull request by jdroenner**

### CODE REVIEW

**Comment by jdroenner** [comment]
[RasterVectorJoin auf docs.geoengine.io beschreiben](https://trello.com/c/E4DRuOgt/1341-rastervectorjoin-auf-docsgeoengineio-beschreiben)

File: src/operators/rastervectorjoin.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+The `RasterVectorJoin` operator allows combining a single vector input and multiple raster inputs.
+For each raster input, a new column is added to the collection from the vector input.
+The new column contains the value of the raster at the location of the vector feature.
+For features covering multiple pixels like `MultiPoints` or `MultiPolygons`, the value is calculated using an aggregation function selected by the user.
+The same is true if the temporal extend of a vector feature covers multiple raster time-steps.
+More details are described below.
+
+**Example**:
+You have a collection of aggricaltural fields (`Polygons`) and a collection of raster images containing the monthly NDVI value for each pixel.
```
Comment (ChristianBeilschmidt) [inline src/operators/rastervectorjoin.md:11]:
```suggestion
You have a collection of agricultural fields (`Polygons`) and a collection of raster images containing each pixel's monthly NDVI value.
```

typo + grammarly suggestion

File: src/operators/rastervectorjoin.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+The `RasterVectorJoin` operator allows you to do this.
```
Comment (ChristianBeilschmidt) [inline src/operators/rastervectorjoin.md:12]: Clarify: to do what?

File: src/operators/rastervectorjoin.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+You now have multiple options to calculate the NDVI value for each field.
+For a quick evaluation, you can use the `first` aggregation function.
```
Comment (ChristianBeilschmidt) [inline src/operators/rastervectorjoin.md:14]: Maybe first state what it is good for in general.
Then say, that it is cheap and also good for exploratory use.

File: src/operators/rastervectorjoin.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    "temporalAggregation": "mean"
+  },
+  "sources": {
+    "vector": {
+      "type": "OgrSource",
+      "params": {
+        "data": {
+          "type": "internal",
+          "datasetId": "e977b123-ca47-4c5b-aace-481119826aaf"
+        }
```
Comment (ChristianBeilschmidt) [inline src/operators/rastervectorjoin.md:61]:
```suggestion
        "data": "ne_10m_ports"
```

File: src/operators/rastervectorjoin.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+      }
+    },
+    "rasters": [
+      {
+        "type": "GdalSource",
+        "params": {
+          "data": {
+            "type": "internal",
+            "datasetId": "b6191257-6d61-4c6b-90a4-ebfb1b23899d"
+          }
```
Comment (ChristianBeilschmidt) [inline src/operators/rastervectorjoin.md:71]:
```suggestion
          "data": "ndvi"
```

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: src/operators/rastervectorjoin.md (hunk 2 above)
Comment (jdroenner) [reply src/operators/rastervectorjoin.md]: [ec1758e](https://github.com/geo-engine/docs/pull/47/commits/ec1758eeb071d2f3f612b9b54b9051f980628b87)

**Review by ChristianBeilschmidt — APPROVED** [review]

**Pull request merged**